src/redux/reducer.js

```diff
@@ -0,0 +1,25 @@
+import { REQUEST_SPENDINGS_SUCCESS } from './types';
+
+const initialState = {
+  spendings: [],
+  pagination: {
+    total: undefined,
+    page: 1,
+    perPage: 5,
+  },
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case REQUEST_SPENDINGS_SUCCESS:
```

Comment:
Why don't you use `REQUEST_SPENDINGS_PENDING` and `REQUEST_SPENDINGS_ERROR` actions here? You can add `loading` filed into `initialState` and show loading screen on your page while request is in process (`REQUEST_SPENDINGS_PENDING`) and hide it when request is finished (`REQUEST_SPENDINGS_SUCCESS`, `REQUEST_SPENDINGS_ERROR`).